jwst/fits_generator/create_dms_data.py

```diff
@@ -3,7 +3,7 @@
 import os
 import numpy
 from astropy.io import fits as pyfits
-import jwst.fits_generator
+from .. import fits_generator
 
```

Comment:
Hmmm, never mind. I think the original intent was to in fact import the entire fits_generator package, hence your original "from .. import fits_generator" is probably correct.